tests/integration_tests/base_tests.py

```diff
@@ -369,9 +369,9 @@ def run_sql(
         ctas_method=CtasMethod.TABLE,
         template_params="{}",
     ):
-        if user_name:
+        if username:
             self.logout()
-            self.login(username=(user_name or "admin"))
+            self.login(username=username)
```

Comment:
Given that on like #372 `username` is truthy there's no need for the `username or "admin"` logic.